packages/cli/lib/project-generator.js

```diff
@@ -131,7 +132,7 @@ module.exports = class ProjectGenerator extends BaseGenerator {
         name: 'name',
         message: 'Project name:',
         when: this.projectInfo.name == null,
-        default: this.options.name || this.appname,
+        default: this.options.name || path.basename(process.cwd()),
```

Comment:
This might be problematic. For example, if I run `lb4` in `/tmp`, would `tmp` be proposed here with the change?

Reply:
If you're in the `/tmp`, then yes the default project name will be `tmp`

Reply:
We don't always need to have a default name, right?

Reply:
I actually don't know why we need the default name for the project. If I were the user, I would be confused if the default name is the folder name TBH. Maybe we can remove the default name for this step?

Reply:
FWIW, the application generator (`lb4 app`) has `applicationName` option, perhaps it was the intention to use that one? IDK.

https://github.com/strongloop/loopback-next/blob/08bf6cc8c1e1b3bfde74c2f6a6c575e6b6b3acb4/packages/cli/generators/app/index.js#L31-L34

I am little bit reluctant to land this change together with other changes in the same commit. Also, is there any easy way how to test this? E.g. create a new directory in the sandbox with a known name, then verify that the generator picked that directory name as the default name.

Reply:
>  is there any easy way how to test this? E.g. create a new directory in the sandbox with a known name, then verify that the generator picked that directory name as the default name.

PTAL ☝️